client/src/components/game/lab2/components/Repair.js

```diff
@@ -0,0 +1,579 @@
+                <div className="code_editor__line--pink">
+                  <span>&nbsp;&nbsp;</span>
+                  <span className='form-error'>One or more of the colors you entered are too dark. Please make
+                  adjustments and submit again.</span>
+                </div>
+                }
+              </div>
+              {/*Correct Circle One*/}
+              <p className="code_editor__class">&#125;</p>
+              <p className="code_editor__class">.correctCircle&#123;</p>
+              <div className="code_editor__form">
+                <div className="code_editor__line">
+                  <span>&nbsp;&nbsp;&nbsp;&nbsp;</span>
+                  <span className="code_editor__line--darkgreen">
+                    &#47;&#47; Adjust this to change the correct color option
+                  </span>
+                </div>
+                <div className="code_editor__property code_editor__line-background--light">
+                  <span>&nbsp;&nbsp;&nbsp;&nbsp;</span>
+                  <span>color:&nbsp;</span>
+                </div>
+                <div className="code_editor__input">
+                {this.state.correctColorPopup ? (
+                        <div className="colorSelector">
+                          <PhotoshopPicker
+                            onChangeComplete={this.onCorrectColorChange}
+                            onAccept={closeCorrectColor}
+                            onCancel={revertCorrectColor}
+                            color={this.state.correctColor}
+                          />
+                        </div>
+                      ) : (
+                        <button
+                          onClick={changeCorrectColor}
+                          style={{ backgroundColor: this.state.correctColor }}
+                          className={`form ${this.state.errorEqual || this.state.errorDarkBackground ? "form-error-input": ""}`}
+                        ></button>
+                      )}
+                </div>
+                { this.state.errorEqual &&
+                <div className="code_editor__line--pink">
+                  <span>&nbsp;&nbsp;</span>
+                  <span className='form-error'>Two of your entered colors are equal to one another. Please make
+                  adjustments and submit again.</span>
+                </div>
+                }
+                { this.state.errorDarkBackground &&
+                <div className="code_editor__line--pink">
+                  <span>&nbsp;&nbsp;</span>
+                  <span className='form-error'>One or more of the colors you entered are too dark. Please make
+                  adjustments and submit again.</span>
+                </div>
+                }
+              </div>
+              {/*Incorrect Circle One*/}
+              <p className="code_editor__class">&#125;</p>
+              <p className="code_editor__class">.incorrectCircleOne&#123;</p>
```

Comment:
The highlighting on the words "color" and "background" under the incorrectCircleOne, incorrectCircleTwo, correctCircle, and home selectors start from the beginning of the line, whereas the highlighting on "display" and "justify-content" under the center selector, and "font-size", "display", and "justify-content" under the header selector start from the beginning of the word 
<img width="340" alt="Screen Shot 2021-03-08 at 5 45 02 PM" src="https://user-images.githubusercontent.com/54957497/110392085-6a016e00-8036-11eb-8d01-669f2fbed183.png">